Google.PowerShell/PubSub/GcpsMessage.cs

```diff
@@ -106,23 +108,11 @@ protected override void ProcessRecord()
         {
             Name = GetProjectPrefixForSubscription(Name, Project);
             PullRequest requestBody = new PullRequest();
-
-            if (MaxMessages.HasValue)
-            {
-                requestBody.MaxMessages = MaxMessages;
-            }
-            else
-            {
-                requestBody.MaxMessages = 100;
-            }
-
-            if (ReturnImmediately.IsPresent)
-            {
-                requestBody.ReturnImmediately = true;
-            }
-            else
+            requestBody.ReturnImmediately = ReturnImmediately.IsPresent;
+            requestBody.MaxMessages = MaxMessages.HasValue ? MaxMessages : DefaultMaxMessages;
+            if (requestBody.MaxMessages < 0)
```

Comment:
Should this throw instead of silently changing the value?  It seems like an exception should be thrown for invalid input. 

Either way can a test be added for this case?